### PR TITLE
fix user is not getting notfied on item rejection

### PR DESCRIPTION
**Pull request by namrataukirde**

### CODE REVIEW

File: app/controllers/review_item/reject.js [hunk 1]
```diff
@@ -46,97 +45,122 @@ export default Ember.Controller.extend({
...
-      this.set('noReasonSelected', true);
+  rejectValidation(selectedReason, rejectProperties) {
+    if (selectedReason === undefined) {
+      this.set("noReasonSelected", true);
       return false;
     }
-    if(selectedReason === "-1" && Ember.$.trim(rejectProperties.rejectReason).length === 0) {
+    if (
+      selectedReason === "-1" &&
+      Ember.$.trim(rejectProperties.rejectReason).length === 0
```
Comment (Darshan-Chauhan) [inline app/controllers/review_item/reject.js]: Do we need to compare with 0 or only length check is sufficient?

File: app/controllers/review_item/reject.js [hunk 2]
```diff
@@ -46,97 +45,122 @@ export default Ember.Controller.extend({
...
-      if(this.get("itemId") && this.cannotSave()){
+      if (this.get("itemId") && this.cannotSave()) {
         return false;
       }
-      var selectedReason = this.get('selectedId');
-      var rejectProperties = this.getProperties('rejectReason');
-      if(!this.rejectValidation(selectedReason, rejectProperties)){
+      var selectedReason = this.get("selectedId");
+      var rejectProperties = this.getProperties("rejectReason");
+      rejectProperties.rejectionComments = Ember.$("#rejectMessage").val();
```
Comment (Darshan-Chauhan) [inline app/controllers/review_item/reject.js]: I would very much prefer to not use jQuery

File: app/controllers/review_item/reject.js (hunk 1 above)
Comment (namrataukirde) [reply app/controllers/review_item/reject.js]: Lol.. Jshint.. Still i will change it. Thanks.

File: app/controllers/review_item/reject.js (hunk 2 above)
Comment (namrataukirde) [reply app/controllers/review_item/reject.js]: Hehe.. Sure.

**Pull request closed without merging**